### DISCUSSION TITLE
Use jank's dashboard to track progress

### DISCUSSION THREAD

**Discussion by simonacca** [squint-cljs/cherry#113] [Ideas]
[Jank's progress dashboard](https://jank-lang.org/progress/) is pretty neat, would be useful for newcomers to cherry to understand how far along things are.

I'd be happy to set it up if there's interest. Also willing to take a stab at filling in data for the current state of affairs ( 
this would need review though).

[Dashboard source](https://github.com/jank-lang/jank-lang.org/blob/main/src/org/jank_lang/page/progress/view.clj)

**Comment by borkdude** [comment]
Thanks for the idea. I will be pretty busy until Strangeloop (21st of September) where I'll be doing a talk, after that I might have a look
